dashboard/app/reporting_email.go

```diff
@@ -454,20 +456,27 @@ func warnMailingListInCC(c context.Context, msg *email.Email, mailingList string
 	}
 }
 
-func sendMailTemplate(c context.Context, subject, from string, to []string, replyTo string,
+func sendMailTemplate(c context.Context, subject, from string, to, cc []string, replyTo string,
 	attachments []aemail.Attachment, template string, data interface{}) error {
 	body := new(bytes.Buffer)
 	if err := mailTemplates.ExecuteTemplate(body, template, data); err != nil {
 		return fmt.Errorf("failed to execute %v template: %v", template, err)
 	}
-	return sendMailText(c, subject, from, to, replyTo, attachments, body.String())
+	return sendMailText(c, subject, from, to, cc, replyTo, attachments, body.String())
 }
 
-func sendMailText(c context.Context, subject, from string, to []string, replyTo string,
+func sendMailText(c context.Context, subject, from string, to, cc []string, replyTo string,
 	attachments []aemail.Attachment, body string) error {
+	if len(to) == 0 {
+		to = email.MergeEmailLists(to, cc)
+		cc = []string{}
+	} else {
+		to, cc = separateCC(to, cc)
```

Comment:
separateCC seems to always return the original to, so reassigning it is excessive and confusing (for me as a reader it suggests that separateCC can change to).
And the previous branch can be shortened to:
```
cc = separateCC(to, cc)
if len(to) == 0 {
	to, cc = cc, nil
}
```